app.py

```diff
@@ -124,13 +120,6 @@
     Route("/notify/{path:path}", Notify),
     Route("/healthcheck", Healthcheck),
     Route("/project/{project_id:str}", ProjectDetails),
```

Comment:
i think some other routes are also not in use anymore. e.g. /import/task/ with UploadTaskInfo was afaik only used in the proxy. Some other might also be irrelevant by now but never were removed. ProjectDetails, CognitionImport & CogntiionPrepareProject since we still have some time lets also remove them where possible.

Reply:
Sounds good, looking into removing them 👍